sample.env

```diff
@@ -3,10 +3,10 @@ ACCESS_TOKEN=token
 REFRESH_TOKEN=short-token
 
 # API URLS (for RERUM instance)
-RERUM_REGISTRATION_URL = https://store.rerum.io/v1/
-RERUM_API_ADDR = https://store.rerum.io/v1/api/
-RERUM_ID_PATTERN = https://store.rerum.io/v1/id/
-RERUM_ACCESS_TOKEN_URL = http://store.rerum.io/client/request-new-access-token
+RERUM_REGISTRATION_URL = https://devstore.rerum.io/v1
```

Comment:
I would make the case that this could be left as "store" because it is a sample for production. At least the access tokens, since it is shared twixt the two may be left as "store".